src/intl/ja/page-layer-2.json

```diff
@@ -57,7 +57,7 @@
   "layer-2-dyor-3": "L2BEATへ移動",
   "layer-2-use-layer-2-title": "レイヤー2を使う",
   "layer-2-use-layer-2-1": "レイヤー2がなぜ必要か、またレイヤー2の仕組みを理解したところで、さっそく使ってみましょう。",
-  "layer-2-contract-accounts": "<strong>注:</strong> ブリッジしてレイヤー2を使用する場合、イーサリアムメインネットと同様、ご自身がEOAアカウント(単一の秘密鍵のみがアカウントを制御するアカウント)のアドレス管理をすることに留意することが重要です。ただし、<a href=\"https://safe.global/\"  target=”_blank”>Safe</a>や<a href=\"https://www.argent.xyz/\"  target=”_blank”>Argent</a>などのコントラクトアカウントを使用している場合は、コントラクトアカウントをレイヤー2上のアドレスに再デプロイするまで、レイヤー2上で、そのアドレスを制御することはできません。ブリッジ、またはコントラクトアカウントへ送金を行う際、コントラクトアカウントのアドレスを制御していない場合は、資金が失われるおそれがあります。",
+  "layer-2-contract-accounts": "<strong>注:</strong> ブリッジしてレイヤー2を使用する場合、イーサリアムメインネットと同様、ご自身がEOAアカウント(単一の秘密鍵のみがアカウントを制御するアカウント)のアドレス管理をすることに留意することが重要です。ただし、<a href=\"https://gnosis-safe.io/\"  target=”_blank”>Gnosis Safe</a>や<a href=\"https://www.argent.xyz/\"  target=”_blank”>Argent</a>などのコントラクトアカウントを使用している場合は、コントラクトアカウントをレイヤー2上のアドレスに再デプロイするまで、レイヤー2上でこのアドレスを制御することはできません。ブリッジ、またはコントラクトアカウントへ送金を行う際、コントラクトアカウントのアドレスを制御していない場合は、資金が失われるおそれがあります。",
```

Comment:
`https://gnosis-safe.io/` is now `https://safe.global/`

Reply:
```suggestion
  "layer-2-contract-accounts": "<strong>注:</strong> ブリッジしてレイヤー2を使用する場合、イーサリアムメインネットと同様、ご自身がEOAアカウント(単一の秘密鍵のみがアカウントを制御するアカウント)のアドレス管理をすることに留意することが重要です。ただし、<a href=\"https://safe.global/\"  target=”_blank”>Safe</a>や<a href=\"https://www.argent.xyz/\"  target=”_blank”>Argent</a>などのコントラクトアカウントを使用している場合は、コントラクトアカウントをレイヤー2上のアドレスに再デプロイするまで、レイヤー2上でこのアドレスを制御することはできません。ブリッジ、またはコントラクトアカウントへ送金を行う際、コントラクトアカウントのアドレスを制御していない場合は、資金が失われるおそれがあります。",
```